.azure-pipelines/all_pr.yml

```diff
@@ -5,6 +5,7 @@ pr:
   paths:
     include:
     - datadog_checks_base/datadog_checks/*
+    - .azure-pipelines/*
```

Comment:
I think it reasonable to trigger `PR All` build when `.azure-pipelines/*` changes.